include/net/nrf_provisioning.h

```diff
@@ -0,0 +1,120 @@
+/*
+ * Copyright (c) 2022 Nordic Semiconductor ASA
```

Comment:
Change the year to 2023 as this is the first year we are going to push this public.
Same goes to all C and H files.

Reply:
Changes done